servers/visual/visual_server_raster.cpp

```diff
@@ -1641,7 +1641,6 @@ void VisualServerRaster::camera_set_perspective(RID p_camera,float p_fovy_degree
 	camera->fov=p_fovy_degrees;
 	camera->znear=p_z_near;
 	camera->zfar=p_z_far;
-
 }
 
 void VisualServerRaster::camera_set_orthogonal(RID p_camera,float p_size, float p_z_near, float p_z_far) {
```

Comment:
The merges in this file are really obsolete as all the code has moved into visual_server_scene but they got merged in from the Godot2 version of this change.